modules/getting-started-openshift-gitops.adoc

```diff
@@ -0,0 +1,128 @@
+
+To find the password for the ArgoCD:
+
+. Go to the table of contents and make sure you are in the *Developer* perspective and not the *Administrator* perspective.
+  *  You now see a list of available projects.
+. Select the *Secrets* tab.
+. Select the *argo-cluster-cluster* instance to reveal the password.
+. Copy the password.
+. Use this password to log into the ArgoCD UI.
+
+*Note:* if you want to fetch the ArgoCD password from the command line interface, run the following command:
+
+ `oc get secret argocd-cluster-cluster -n openshift-gitops -o jsonpath='{.data.admin\.password}' | base64 -d`
+
+[id="setting-up-a-new-argocd-instance_{context}"]
+== Setting up a new ArgoCD instance
+
+The Red Hat OpenShift GitOps operator automatically installs an instance of ArgoCD in the "openshift-gitops" namespace with additional permissions, allowing ArgoCD to manage cluster-scoped resources.
+
+If you want to install your own instance for managing your cluster configurations or deploying your applications, you must deploy your own instance of ArgoCD. Using your own instance of ArgoCD helps you to continuously deliver your applications.
+
+.Installing ArgoCD
+
+To install an ArgoCD instance:
+
+. Go to the table of contents and make sure you are in the *Administrator* perspective and not the *Developer* perspective.
+. Select *Operators* then select *Installed Operators*.
+. Create a new project or select an existing project, depending on where you want to install the ArgoCD instance.
+. Select *OpenShift GitOps* from the "Installed Operators" page.
+. Select the *ArgoCD* tab.
+. Create an ArgoCD instance and specify the following configurations:
+  * Name: <argocd-instance-name>
+  * Server > Route > Enable Route = true. This configuration creates an external OS route to access the ArgoCD server.
+
+.Deploying resources to a different namespace
+
+To grant ArgoCD the permissions to manage resources in multiple namespaces, create RoleBindings which provide ArgoCD admin permissions for that particular namespace. Use the example, OOTB ArgoCD instance, deployed in the openshift-gitops namespace.
+
+To create a RoleBinding:
+
+. Go to the table of contents and make sure you are in the *Administrator* perspective and not the *Developer* perspective.
+. Go to *User Management* > *Role Bindings* > select *Create Binding*.
+  * You now see a form that guides you through the process of creating a RoleBinding:
+. For *Binding Type*, select *Namespace Role Binding (RoleBinding)*.
+  * If you only select *All Projects*, then you only need to provide the *Binding Type*.
+  * If the project is already selected as the one you want to create the RoleBinding for, the project does not ask for *Binding Type* because it assumes it is a RoleBinding.
```

Comment:
Probably combine content on line 67 and 68 something like this: 
You need to provide the *Binding Type* only when you select *All Projects*.  You do not need to specify the binding type if a particular project is selected, as the type is assumed to be role binding for the project.